OpenSim/Sandbox/MatlabScripts/opensimTimeSeriesTableToMatlab.m

```diff
@@ -1,12 +1,23 @@
 function data = opensimTimeSeriesTableToMatlab(table)
 
 
+%%
+import org.opensim.modeling.*
+%%
 nCol = table.getNumColumns;
 nRow = table.getNumRows;
 
+%% 
 data = struct;
-rowdata = zeros(nRow, 3);
 
+%%
+if strcmp( char(table.getClass), 'class org.opensim.modeling.TimeSeriesTableVec3')
+    rowdata = zeros(nRow, 3);
+elseif strcmp( char(table.getClass), 'class org.opensim.modeling.TimeSeriesTable')
+    rowdata = zeros(nRow, 1);
+end
```

Comment:
You may want to handle more than just these two types of tables. Consider using `flatten()` first, so you can handle all tables uniformly.